controllers/cartController.js

```diff
@@ -0,0 +1,20 @@
+const cartService = require("../services/cartService");
+const { asyncErrorHandler } = require("../middlewares/errorHandling");
+
+// 2. 장바구니 상품 조회
+const readCart = asyncErrorHandler(async (req, res) => {
+  // JWT 로부터 추출한 user id
+  const userId = req.userId;
+  if (!userId) {
+    const err = new Error("PARAMS_ERROR");
+    err.statusCode = 400;
+    throw err;
+  }
+  cartList = await cartService.readCart(userId);
+  return res.status(201).json(cartList);
+  // return res.status(201).json({ data: cartList });
```

Comment:
- 상황에 맞는 status code를 사용해 주세요.
- 불필요한 주석은 제거해 주세요.
```suggestion
  return res.status(200).json(cartList);
```